text.cabal

```diff
@@ -94,8 +94,12 @@ library
       extra-libraries: c++
     elif os(openbsd)
       extra-libraries: c++ c++abi pthread
-    else
+    elif os(linux) || os(windows)
       extra-libraries: stdc++
+    else
```

Comment:
Please add a comment, explaining that this clause is supposed to be used with `wasm-wasi`, but you cannot spell it explicitly because of `cabal check`. Add a link to the relevant Cabal ticket.